src/modules/layout/containers/HeaderContainer.js

```diff
@@ -2,6 +2,23 @@
 
 import Header from '../components/Header'
 import { withTheme } from 'styled-components/native'
+import type { StateType } from '../../app/StateType'
+import { connect } from 'react-redux'
+import React from 'react'
+import { withNavigation } from 'react-navigation'
 
-// $FlowFixMe
-export default withTheme(Header)
+const mapStateToProps = (state: StateType, ownProps) => {
+  const key: string = ownProps.navigation.getParam('key')
+  // todo add availableLanguages for other routes
```

Comment:
Not quite sure if I should do this in this task, since there is https://issues.integreat-app.de/browse/NATIVE-180 .

Reply:
This should work for events and categories for now.